CODEOWNERS

```diff
@@ -3,7 +3,9 @@
 
 # inventory.json schemas, main logic and tests
 /json-lint/schemas/inventory/ @Marfeel/alot
+/json-lint/schemas/inventory/placements/placement.json @albertmolinermrf @adriafolchmrf
```

Comment:
Can this be also the entire ALOT?

Reply:
@ettoreciprianmrf This is provisional because we had to duplicate `placement.json` since we still are not able to share between twister schema and inventory schema.

We are waiting for @davidwasamrf to help us on this